### PR TITLE
Add CLI with colors

### LINKED ISSUE #1: Setup CLI with color
Zypper has color output. This isn't unusual in modern CLI interfaces. The CLI will often try to detect if colors can be used and only use them when it will work. Along with this there is often an opt-out. Colors, such as red, can make it easier to spot errors.

This issue is to add that capability to `cmd/hypper`. A 3rd party library should be used for this.

### PR DESCRIPTION
**Pull request by dragonchaser**
This PRs adds a basic CLI skel with colors to hypper.

Closes #1

### CODE REVIEW

File: cmd/require/args.go [hunk 1]
```diff
@@ -0,0 +1,88 @@
+/*
+Copyright The Helm Authors.
```
Comment (Itxaka) [inline cmd/require/args.go:2]: headers should be changed unless this is directly lifted from helm?

**Review by Itxaka — APPROVED** [review]

File: cmd/require/args.go (hunk 1 above)
Comment (dragonchaser) [reply cmd/require/args.go:2]: Yes, we have an open issue about license headers #8 it is being looked into, will fix as soon this has been clarified.

Comment (Itxaka) [reply cmd/require/args.go:2]: :+1:

Comment (viccuad) [reply cmd/require/args.go:2]: I would drop it, considering that it all is boilerplate code and hardly copyrightable.

Comment (viccuad) [reply cmd/require/args.go:2]: I would even drop everything in `require/args*`, I don't think we are using it for now.

File: cmd/hypper/hypper.go [hunk 2]
```diff
@@ -1,7 +1,42 @@
...
+	"os"
+
+	"github.com/fatih/color"
+	"github.com/mattfarina/hypper/pkg/cli"
+	"github.com/spf13/cobra"
+)
+
+var settings = cli.New()
+
+var red = color.New(color.FgRed).SprintFunc()
```
Comment (Itxaka) [inline cmd/hypper/hypper.go:15]: you can use `//nolint:golint,unused` here I think to silence the linter for now

**Review by Itxaka — APPROVED** [review]

**Pull request merged**